### DISCUSSION TITLE
Compilation on Mac

### DISCUSSION THREAD

**Discussion by accessv** [dail8859/NotepadNext#631] [Q&A]
Hi,
I am trying to figure out how to compile the app on the Mac. Any guides or just recommendations are very welcome .
Regards,
Dan

**Comment by dail8859** [comment]
To be honest, I'm not quite sure. MacOS support was added by other individuals. However you can take a look at the https://github.com/dail8859/NotepadNext/blob/master/.github/workflows/build.yml file to see what it does to build this application for MacOS but I'm not sure what all prerequisites are already there.

If you want to see more specifics of the workflow in action you can take a look at the action output here: https://github.com/dail8859/NotepadNext/actions/runs/10434698073/job/28897845499

If you do come up with any details feel free to create a PR and update the https://github.com/dail8859/NotepadNext/blob/master/doc/Building.md file.
